server/windows/src/windows.rs

```diff
@@ -65,7 +65,7 @@ mod omsupply_service {
                 return;
             }
         };
-        logging_init(settings.logging.clone(), None);
+        logging_init(settings.logging.clone(), None, true);
```

Comment:
Was wondering, why this one is 'update'

Reply:
Nice catch XD [cbd8bf8](https://github.com/openmsupply/open-msupply/pull/1921/commits/cbd8bf8502926e3be1be0cf5161f62a0590869a7)